liota/dcc/dcc_base.py

```diff
@@ -53,4 +53,8 @@ def subscribe(self):
         pass
 
     def create_metric(self, gw, details, unit, sampling_function, sampling_interval_sec=10, aggregation_size=6):
+        self.publish_unit(gw, details, unit)
```

Comment:
Maybe, but that would require a change to the interface because optional parameters cannot go before mandatory ones. I would say, that sounds reasonable but probably not a high priority one now.